packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.Props.ts

```diff
@@ -75,6 +75,21 @@ export interface ICheckboxProps extends React.ButtonHTMLAttributes<HTMLElement |
   theme?: ITheme;
 
   /**
+   * Accessibile label for the checkbox.
+   */
+  ariaLabel?: string;
+
+  /**
+   * ID for element that contains label information for the checkbox. By default, passed-in 'label' property with be used.
```

Comment:
"with" -> "will"?
This description is still confusing, I would just leave off the second sentence.